Bookcrossing.Contracts/Abstractions/RequestFeatures/RequestFeatures.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Bookcrossing.Contracts.Enumeration;
+
+namespace Bookcrossing.Contracts.Abstractions.RequestFeatures
+{
+    public abstract class RequestFeatures
+    {
```

Comment:
RequestFeatures is not good name just because it's not enough discriptive.

Reply:
Is all of you SomethingParams have SearchTerm? If so, why you didn't add it to base class?